addons/point_of_sale/models/pos_config.py

```diff
@@ -810,13 +810,14 @@ def update_customer_display(self, order, access_token):
 
     def _get_customer_display_data(self):
         self.ensure_one()
+        display_device = self.iot_device_ids.filtered(lambda d: d.type == 'display')
```

Comment:
in community this should be just `self.proxy_ip`.
Maybe we can have something like `_get_customer_display_proxy_ip(self)` ( like we have for the frontend )

Reply:
I introduced `_get_display_device_ip` which is overridden in pos_iot.